<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>std::vector demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="20">
     <widget class="QListWidget" name="list_widget"/>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="txt_elem_content"/>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="pb_edit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QPushButton" name="pb_find">
      <property name="text">
       <string>find</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="le_count"/>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="pb_count">
      <property name="text">
       <string>count</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="lbl_count">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Методы</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QPushButton" name="btn_pop_front">
      <property name="text">
       <string>pop_front</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="3">
     <widget class="QPushButton" name="btn_push_front">
      <property name="text">
       <string>push_front</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QPushButton" name="btn_pop_back">
      <property name="text">
       <string>pop_back</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="3">
     <widget class="QPushButton" name="btn_push_back">
      <property name="text">
       <string>push_back</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QPushButton" name="btn_erase">
      <property name="text">
       <string>erase</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QPushButton" name="btn_insert">
      <property name="text">
       <string>insert</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="3">
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Алгоритмы</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <widget class="QPushButton" name="pb_min">
      <property name="text">
       <string>min_element</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="3">
     <widget class="QPushButton" name="pb_max">
      <property name="text">
       <string>max_element</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="3">
     <widget class="QPushButton" name="pb_sort">
      <property name="text">
       <string>merge sort</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4" colspan="3">
     <widget class="QPushButton" name="pb_SORT">
      <property name="text">
       <string>merge sOrT</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="3">
     <widget class="QPushButton" name="pb_unique">
      <property name="text">
       <string>unique</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4" colspan="3">
     <widget class="QPushButton" name="pb_reverse">
      <property name="text">
       <string>reverse</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="3">
     <widget class="QPushButton" name="pb_snuffle">
      <property name="text">
       <string>shuffle</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1" colspan="3">
     <widget class="QPushButton" name="btn_lower_bound">
      <property name="text">
       <string>lower_bound</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4" colspan="3">
     <widget class="QPushButton" name="btn_upper_bound">
      <property name="text">
       <string>upper_bound</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="1" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Итератор</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1" colspan="3">
     <widget class="QPushButton" name="btn_dec_iterator">
      <property name="text">
       <string>--</string>
      </property>
     </widget>
    </item>
    <item row="15" column="4" colspan="3">
     <widget class="QPushButton" name="btn_inc_iterator">
      <property name="text">
       <string>++</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1" colspan="3">
     <widget class="QPushButton" name="btn_begin">
      <property name="text">
       <string>begin</string>
      </property>
     </widget>
    </item>
    <item row="16" column="4" colspan="3">
     <widget class="QPushButton" name="btn_end">
      <property name="text">
       <string>end</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Заготовки</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1" colspan="3">
     <widget class="QPushButton" name="btn_tea">
      <property name="text">
       <string>= чай</string>
      </property>
     </widget>
    </item>
    <item row="18" column="4" colspan="3">
     <widget class="QPushButton" name="btn_cakes">
      <property name="text">
       <string>= торты</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>size():</string>
      </property>
     </widget>
    </item>
    <item row="19" column="2" colspan="3">
     <widget class="QLineEdit" name="txt_size">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="19" column="5" colspan="2">
     <widget class="QPushButton" name="pb_resize">
      <property name="text">
       <string>resize</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>